### PR TITLE
Update STMP and PTMP settings in host file for Orion and Hercules 

### PR DESCRIPTION
**Pull request by TerrenceMcGuinness-NOAA**
# Description

- Updating STMP and PTMP settings in host file for Orion and Hercules because they are cross mounted.
- Also took the opportunity to finally update **SLURM_ACCOUNT** to **HPC_ACCOUT** in CI over rides.
- Added a  refactor of the  `rocotostat.py` tool that is more pythonic and as a execute retry feature because the `rocotostat` utility on Orion has been failing sometimes.

# Type of change
- Bug fix (fixes something broken)

# Change characteristics
- Is this a breaking change (a change in existing functionality)? NO
- Does this change require a documentation update? NO

### CODE REVIEW

File: ci/platforms/config.hercules [hunk 1]
```diff
@@ -4,6 +4,6 @@ export GFS_CI_ROOT=/work2/noaa/stmp/GFS_CI_ROOT/HERCULES
 export ICSDIR_ROOT=/work/noaa/global/glopara/data/ICSDIR
 export STMP="/work2/noaa/stmp/${USER}/HERCULES"
 export PTMP="/work2/noaa/stmp/${USER}/HERCULES"
```
Comment (DavidHuber-NOAA) [inline ci/platforms/config.hercules]: These should be removed.

File: ci/platforms/config.wcoss2 [hunk 2]
```diff
@@ -4,6 +4,6 @@ export GFS_CI_ROOT=/lfs/h2/emc/global/noscrub/globalworkflow.ci/GFS_CI_ROOT
 export ICSDIR_ROOT=/lfs/h2/emc/global/noscrub/emc.global/data/ICSDIR
 export STMP="/lfs/h2/emc/stmp/${USER}"
 export PTMP="/lfs/h2/emc/ptmp/${USER}"
```
Comment (DavidHuber-NOAA) [inline ci/platforms/config.wcoss2]: These should be removed.

**Review by DavidHuber-NOAA — CHANGES_REQUESTED** [review]

File: ci/platforms/config.hercules (hunk 1 above)
Comment (TerrenceMcGuinness-NOAA) [reply ci/platforms/config.hercules]: Yes, just as where on Orion, not sure how that got missed. Thanks.

File: ci/platforms/config.wcoss2 (hunk 2 above)
Comment (TerrenceMcGuinness-NOAA) [reply ci/platforms/config.wcoss2]: Yes.

**Review by WalterKolczynski-NOAA — APPROVED** [review]
Conditionally approved pending successful completion of CI tests.

**Comment by DavidHuber-NOAA** [comment]
Looks like Orion's rocoto server glitched like it did for Hera a few days ago.

**Comment by TerrenceMcGuinness-NOAA** [comment]
Everything looked ok in place.  Made some changes in wait times and added a bug fix on STALL check and retrying.

**Review by DavidHuber-NOAA — APPROVED** [review]
Looks great @TerrenceMcGuinness-NOAA.  This is going to help PRs a lot.

**Pull request merged**